lib/validators.rb

```diff
@@ -8,129 +10,119 @@
     end
+  end
 
-    class Cat1R31 < Schematron::Validator
-      include Singleton
-
-      def initialize
-        super("QRDA Cat 1 Validator", File.join(BASE_DIR, QRDA_CAT1_R3_1_SCHEMATRON))
-      end
+  class Cat1R31 < Schematron::Validator
+    include Singleton
 
+    def initialize
+      super('QRDA Cat 1 Validator', File.join(BASE_DIR, QRDA_CAT1_R3_1_SCHEMATRON))
     end
+  end
 
-    class Cat1R4 < Schematron::Validator
-      include Singleton
-
-      def initialize
-        super("QRDA Cat 1 Validator", File.join(BASE_DIR, QRDA_CAT1_R4_SCHEMATRON))
-      end
+  class Cat1R4 < Schematron::Validator
+    include Singleton
 
+    def initialize
+      super('QRDA Cat 1 Validator', File.join(BASE_DIR, QRDA_CAT1_R4_SCHEMATRON))
     end
+  end
 
-    class Cat1R5 < Schematron::Validator
-      include Singleton
-
-      def initialize
-        super("QRDA Cat 1 Validator", File.join(BASE_DIR, QRDA_CAT1_R5_SCHEMATRON))
-      end
+  class Cat1R5 < Schematron::Validator
+    include Singleton
 
+    def initialize
+      super('QRDA Cat 1 Validator', File.join(BASE_DIR, QRDA_CAT1_R5_SCHEMATRON))
     end
+  end
 
-    class Cat3 < Schematron::Validator
-      include Singleton
+  class Cat3 < Schematron::Validator
+    include Singleton
 
-      def initialize
-        super("QRDA Cat 3 Validator", File.join(BASE_DIR, QRDA_CAT3_SCHEMATRON))
-      end
+    def initialize
+      super('QRDA Cat 3 Validator', File.join(BASE_DIR, QRDA_CAT3_SCHEMATRON))
     end
+  end
 
-    class Cat3R11 < Schematron::Validator
-      include Singleton
+  class Cat3R11 < Schematron::Validator
+    include Singleton
 
-      def initialize
-        super("QRDA Cat 3 Validator", File.join(BASE_DIR, QRDA_CAT3_1_1SCHEMATRON))
-      end
+    def initialize
+      super('QRDA Cat 3 Validator', File.join(BASE_DIR, QRDA_CAT3_1_1SCHEMATRON))
     end
+  end
 
-    class Cat3R2 < Schematron::Validator
-      include Singleton
+  class Cat3R2 < Schematron::Validator
+    include Singleton
 
-      def initialize
-        super("QRDA Cat 3 Validator", File.join(BASE_DIR, QRDA_CAT3_2SCHEMATRON))
-      end
+    def initialize
+      super('QRDA Cat 3 Validator', File.join(BASE_DIR, QRDA_CAT3_2SCHEMATRON))
     end
+  end
 
-    class Cat3R21 < Schematron::Validator
-      include Singleton
+  class Cat3R21 < Schematron::Validator
+    include Singleton
 
-      def initialize
-        super("QRDA Cat 3 Validator", File.join(BASE_DIR, QRDA_CAT3_21SCHEMATRON))
-      end
+    def initialize
+      super('QRDA Cat 3 Validator', File.join(BASE_DIR, QRDA_CAT3_21SCHEMATRON))
     end
+  end
 
-    class Cat3PerformanceRate < PerformanceRateValidator
-      include Singleton
-
-      #def initialize
-      #  super("Performance Rate Validator", File.join(BASE_DIR, CDA_SDTC_SCHEMA))
-      #end
+  class Cat3PerformanceRate < PerformanceRateValidator
+    include Singleton
 
-    end
+    # def initialize
```

Comment:
Recommend adding comment explaining why this is commented out

Reply:
Honestly I don't know why this is commented out. That is a direct port from HDS. Probably could just remove this, rather than leave it commented.